d2l-button-subtle.html

```diff
@@ -51,7 +51,7 @@
 				transition: color 0.3s ease;
 				vertical-align: middle;
 			}
-			.d2l-button-subtle-icon {
+			d2l-icon.d2l-button-subtle-icon {
```

Comment:
Is the `d2l-icon` just needed for the Galen tests?

Reply:
that's what I added it for, but it might not actually solve the problem. `padding-right` isn't getting applied, but I believe it's only on edge and I can't reproduce the problem locally.

Reply:
It might be related to the `display` property for `d2l-icon`... ?  Something to maybe look at.

Reply:
Is the purpose of that padding to provide space between the icon and the text?  If so, left/right margin would be another way to go.